### PR TITLE
Remove arguments

### LINKED ISSUE #128: Remove Most Arguments
As seen in Release 0.3, we will be discontinuing arguments very soon. This is due to them creating a hassle where anyone who wants to implement a feature or option basically has to do the config part of it twice, which is not fun.

We will be keeping --config and --dryrun because those are the only two options that cannot be changed inside a config. If you wish to run multiple instances , all you need to do is run it with several configs instead of from a huge unorganized terminal line.

This change will happen very soon.

### PR DESCRIPTION
**Pull request by Evanito**
## Description

Removed all arguments that can be configured in a cfg file, left ``--config`` and ``--dryrun``

Closes #128 

This is to make it easier to add new features in the future by not requiring contributors to program adding the configuration logic for the feature twice. 
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] **I have read CONTRIBUTING.md**
- [x] **I fully understand [Github Flow.](https://guides.github.com/introduction/flow/)**
- [x] **My code adheres to the [code style of this project.](https://poloniexlendingbot.readthedocs.io/en/latest/contributing.html)**
- [x] **I have updated the documentation in /docs if I have changed the config, arguments, logic in how the bot works, or anything that understandably needs a documentation change.**
- [x] **If my change requires a change to the config, I have updated the config file and command line arguments accordingly.**
- [ ] **I have tested the bot with no issues for 24 continuous hours. If issues were experienced, they have been patched and tested again.**

Removed arguments but made sure the remaining logic was the same.

### CODE REVIEW

**Comment by Evanito** [comment]
I went back and disabled checking variable name conventions until we get them worked out, so Codacy shouldn't do this again.

File: lendingbot.py [hunk 1]
```diff
@@ -221,15 +121,10 @@ def timestamp():
 log = {}
 
 # check if json output is enabled
-jsonOutputEnabled = (config.has_option('BOT', 'jsonfile') and config.has_option('BOT', 'jsonlogsize')) or (
-    args.jsonfile and args.jsonlogsize)
+jsonOutputEnabled = (config.has_option('BOT', 'jsonfile') and config.has_option('BOT', 'jsonlogsize'))
```
Comment (rnevet) [inline lendingbot.py]: valid issue.

File: lendingbot.py [hunk 2]
```diff
@@ -15,187 +15,87 @@
...
+if config.has_option("BOT", "coinconfig"):
+    try:
+        # parsed
+        coinconfig = (json.loads(config.get("BOT", "coinconfig")))
+        for cur in coinconfig:
+            cur = cur.split(':')
+            coincfg[cur[0]] = dict(minrate=(Decimal(cur[1])) / 100, maxactive=Decimal(cur[2]),
+                                   maxtolent=Decimal(cur[3]), maxpercenttolent=(Decimal(cur[4])) / 100,
+                                   maxtolentrate=(Decimal(cur[5])) / 100)
+    except Exception as e:
```
Comment (rnevet) [inline lendingbot.py]: we can consider reducing min variable length to 1 (or renaming Exceptions to ex)
To reduce min variable name to 1:
[a-z_][a-z0-9_]{2,30}$ -> [a-z_][a-z0-9_]{*1*,30}$

**Comment by rnevet** [comment]
image
Basically this regex needs to be change to lower cased:
[a-z_][a-z0-9_]{2,30}$

Otherwise it lgtm, needs a testing period.

File: lendingbot.py (hunk 2 above)
Comment (Evanito) [inline lendingbot.py]: I am not opposed to reducing minimum length to 1, I just followed Mik's exception style

**Comment by Evanito** [comment]
LGTM

**Comment by rnevet** [comment]
Lgtm

**Pull request merged**